package.json

```diff
@@ -15,9 +15,8 @@
     "grunt-cli": "0.1.6",
     "grunt-contrib-watch": "~0.5.3",
     "load-grunt-tasks": "~0.2.0",
-    "grunt-ts": "~1.12.1",
-    "grunt-tslint": "2.2.0-beta",
-    "tslint": "2.2.0-beta",
```

Comment:
I think this will break. `grunt-tslint` does not specify `tslint` as a dependency, but rather as a dev dependency, so I doubt it will download the dependency. Can you please double check
```bash
rm -rf node_modules && npm install ^^ grunt tslint
````